api/src/main/resources/liquibase-update-to-2.1.xml

```diff
@@ -279,10 +279,28 @@
 		<addUniqueConstraint tableName="cohort_member" columnNames="uuid"/>
 	</changeSet>
 	<changeSet id="201610242135-1" author="vshankar">
+		<preConditions>
+			<not>
+				<customPrecondition className="com.ctp.liquibase.ForeignKeyExistsPrecondition">
```

Comment:
liquibase has a `<foreignKeyConstraintExists />` that you should use, it's good to use the standard features from a library